Improve prefetch docs (#7692)
zeit/next.js: packages/next/README.md

#### With `<Link>`

> `<Link>` will automatically prefetch pages in the background as they appear in the view. If you don't want to prefetch one or multiple pages you can manually set `prefetch` to `false`, here's how:

Reviewer: ```suggestion
`<Link>` will automatically prefetch pages in the background as they appear in the view. If certain pages are rarely visited you can manually set `prefetch` to `false`, here's how:
```

Author: Thank you 🥇
